FrontEnd/src/Components/SideNav/SideNavFramework.jsx

```diff
@@ -58,6 +59,15 @@ export default class SideNavFramework extends React.Component {
                     onClick={() => window.location.href = logout.link}
                 >
                     {logout.title}
+                </SirioButton>;
+            changePasswordButton =
```

Comment:
Note to self: mungkin penempatan tombol change password jangan langsung dari navbar